res/values-pt-rBR/strings.xml

```diff
@@ -1129,8 +1129,8 @@ Esta é uma lista de coisas que é * PROIBIDO * para usar nas skins personalizad
     <string name="EXCLUSIVE_skins_pets_hats_and_more_">Skins, Masc, Bonés EXCLUSIVOS e Mais!</string>
     <string name="BUY_NOW_">COMPRAR AGORA!</string>
     <string name="already_purchased">Você ou a pessoa para qual você está comprando já possui isso.</string>
-    <string name="Halo_Animations">Animações Aréola</string>
-    <string name="Trick_Notifications">Notificações Trick</string>
+    <string name="Halo_Animations">Animações daAréola</string>
```

Comment:
Missing a space between "da" and "aréola"

Reply:
Yeah, that's what I noticed too during the review. And also, I missed the "dos" in "Margem Botões"